GithubIssueTagger/Program.cs

```diff
@@ -9,8 +11,10 @@ namespace GithubIssueTagger
     public class Program
     {
         private static IList<Issue> _unprocessedIssues;
-        private static IReadOnlyList<Label> _allLabels;
-        private static IEnumerable<Issue> _allIssues;
+        private static IReadOnlyList<Label> _allHomeLabels;
```

Comment:
I don't want to add more to work for, but these statics make me cringe :) 

It'd be neater if you were using return values/passed collections etc.

Reply:
This change really has nothing to do with the statics, but I simply had to split them into 2 collections. Someone can separately PR refactoring if we think that's needed.

Reply:
Yeah, more of a general comment. 

I might do the refactoring opportunistically if you don't mind.